client/app/account/billing/autoRenew/warnPaymentMean/billing-autoRenew-warnPaymentMean.html

```diff
@@ -0,0 +1,10 @@
+<div data-ng-controller="Billing.controllers.AutoRenew.WarnPaymentMean as $ctrl">
+    <oui-modal data-heading="{{ ::'autorenew_service_active_title' | translate }}"
```

Comment:
```suggestion
    <oui-modal data-heading="{{:: 'autorenew_service_active_title' | translate }}"
```

Choose a convention and stick to it :)